app/views/unattended/provisioning_templates/snippet/ansible_tower_callback_script.erb

```diff
@@ -9,10 +9,12 @@ description: |
 -%>
 #!/bin/sh
 
+<% base_api = host_param('ansible_tower_api_url') || "#{host_param('ansible_tower_fqdn')}/api/controller/v2" %>
+
```

Comment:
I'd suggest to keep the whitespace change to a minimum. You can see the difference in the generated snapshots.
```suggestion
<% base_api = host_param('ansible_tower_api_url') || "#{host_param('ansible_tower_fqdn')}/api/controller/v2" -%>
```